src/models/UsersModel.js

```diff
@@ -0,0 +1,51 @@
+class UsersModel {
+    constructor({
+      name,
+      email,
+      profilePicture = 'http://localhost:8000/images/userProfileImage.jpg', // Default image URL
+      loginType = null,
+      adminCode = null,
+      createdAt = new Date(),
+      lastLogin = null,
```

Comment:
Why do we care about lastLogin and lastLogout?

Reply:
You’re right, Eli; we don’t need lastLogin and lastLogout times. I initially thought we could track how long the user is accessing the learning app, but I think it’s better to send an autogenerated mail whenever the user logs in.